source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -246,6 +247,10 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
     serviceSend({ type: 'FINISH_SELECTION' });
   }, [serviceSend]);
 
+  const changeSidebar = useCallback(() => {
```

Comment:
unrelated change, but subdivision/consolidations were not clearing the list of draft markers after the subdivision is saved.